playbooks/common/openshift-cluster/upgrades/pre/validate_excluder.yml

```diff
@@ -27,3 +30,5 @@
     - "{{ excluder_version.stdout.split('.')[0:2] | join('.') | version_compare(openshift_upgrade_target.split('.')[0:2] | join('.'), '>', strict=True) }}"
   when:
   - not openshift.common.is_atomic | bool
+  # validate excluders only if at least one of them is enabled
+  - (enable_excluders | default(false)) or (enable_docker_excluder | default(false)) or (enable_openshift_excluder | default(false))
```

Comment:
Should we not make `(enable_excluders | default(true))` since we also default it to true in the role?